app/models/user.rb

```diff
@@ -1,5 +1,13 @@
 class User < ApplicationRecord
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: Relationship.name,
+                                  foreign_key: "follower_id",
```

Comment:
```suggestion
                                  foreign_key: :follower_id,
```
sao phải có class_name và foreign_key

Reply:
model active_relationships không tồn tại nên ta cần class_name và foreign_key để chỉ rõ tên bảng mà nó tham chiếu đến và khóa ngoài của nó